### PR TITLE
Update tags to latest release

### PR DESCRIPTION
**Pull request by SAY-droid427**
Added code to update the tags  to the latest release. 

Related to https://github.com/adoptium/aqa-tests/issues/1971

### CODE REVIEW

**Comment by llxia** [comment]
Could you provide sample input and output of the running this script? Thanks

**Comment by llxia** [comment]
Could you provide sample input and output of the running this script? Thanks

**Comment by SAY-droid427** [comment]
@llxia  This is the video of the working of this script.
https://drive.google.com/file/d/1Nhprw-5GFi013vwKcsBNBUuTV1c3zA3H/view?usp=sharing
Thanks!

File: .github/workflows/updateToLatestRelease.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import requests
+import json
+
+filepath=input("Enter the file path: ")
```
[the remark this answers is not in the document]
Comment (SAY-droid427) [reply .github/workflows/updateToLatestRelease.py:4]: OK I will make the changes.

File: .github/workflows/updateToLatestRelease.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+modified_repos = []
+no_releases = []
+path = "https://api.github.com/repos/"
+updateflag = False
+
+with open(filepath) as fhand:
+  line = fhand.readline()
+  cnt = 0
```
Comment (sophia-guo) [inline .github/workflows/updateToLatestRelease.py:12]: what's the `cnt` for

File: .github/workflows/updateToLatestRelease.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        elem = request_url+"\n"
+        no_releases.append(elem)
+
+    modified_repos.append(line)
+    line = fhand.readline()
+    cnt += 1
+fhand.close()
+
+# printing the urls for which latest/release donot exist
+print("No latest/release for url:")
```
Comment (sophia-guo) [inline .github/workflows/updateToLatestRelease.py:46]: May be more clearer put those no latest one under the condition of ` if no_releases in not empty`

**Comment by sophia-guo** [comment]
What's the input? One test.properties file or all test.properties files under specific directory?

File: .github/workflows/updateToLatestRelease.py (hunk 2 above)
Comment (SAY-droid427) [reply .github/workflows/updateToLatestRelease.py:12]: This can be removed as it is not being used. I was using it while debugging. Will remove that.

**Comment by SAY-droid427** [comment]
> What's the input? One test.properties file or all test.properties files under specific directory?
> 
> Suppose this is only for repo with releases?
The input is test.properties of a particular repo we want to check for latest release. Do we want to check it like that or any other way?

**Comment by smlambert** [comment]
The way I read what @sophia-guo is asking is whether you want this script to take a single test.properties file as input (as in feed this script (https://github.com/adoptium/aqa-tests/blob/master/external/netty/test.properties), or whether you could input https://github.com/adoptium/aqa-tests/blob/master/external and look in all subdirectories for all test.properties files.

I think if we intend to run this regularly to check if there is anything that needs updating, we may want the latter case, but in order to work for all files, you need to make sure it works first for a single file, which is what I believe you are doing here, correct?

**Comment by SAY-droid427** [comment]
@sophia-guo @smlambert @llxia  I have updated the logic to take command-line input of https://github.com/adoptium/aqa-tests/blob/master/external and search for all the test.properties under that and update them simultaneously in one run. I will attach a working demo in a while because the API rate limits has exceeded for my machine. Looking forward to your review. Thank you!

File: .github/workflows/updateToLatestRelease.py (hunk 3 above)
Comment (SAY-droid427) [reply .github/workflows/updateToLatestRelease.py]: Done

**Review by sophia-guo — APPROVED** [review]
This is the first step of automatically updating latest releases. LGTM.

**Review by llxia — APPROVED** [review]
LGTM. We can enhance this in future PR.

**Pull request merged**